Dockerfile.dapper

```diff
@@ -39,6 +39,12 @@ ENV DOCKER_URL_amd64=https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 \
 
 RUN wget -O /usr/bin/docker ${!DOCKER_URL} && chmod +x /usr/bin/docker
 
+# Minio
+ENV MINIO_URL_amd64=https://dl.minio.io/server/minio/release/linux-amd64/minio \
+    MINIO_URL_arm=https://dl.minio.io/server/minio/release/linux-arm/minio \
```

Comment:
Don't worry about ARM. We don't really support it.

Also can we pin minio to a certain version? Rather than getting latest every time, which may result in incompatible API or bugs later.

Reply:
No. According to minio documents, we can only get Minio server from this url.

Reply:
In fact you can get `https://dl.minio.io/server/minio/release/linux-amd64/archive/minio.RELEASE.2018-05-25T19-49-13Z`

See: https://dl.minio.io/server/minio/release/linux-amd64/archive/